README.adoc

```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
-           "default server": "available"
-        },
-        "name": "SystemResource",
-        "state": "UP"
-      }
-   ],
-   "outcome": "UP"
-}
-----
-
-When you're done checking out the application, stop the Open Liberty server with the following
-command:
+When you are done checking out the services, stop the Open Liberty server by running the following command:
 
 ```
 mvn liberty:stop-server
 ```
 
-Navigate back to the `start` directory to begin.
-
 // =================================================================================================
 // Adding health checks to microservice
 // =================================================================================================
 
 == Adding health checks to microservices
 
-The MicroProfile Health API was added as a dependency to your `pom.xml` file. Look for the dependency with the
-`microprofile-health-api` group ID. This feature allows you to use the MicroProfile Health API to provide health checks
-for your microservices. The `mpHealth-1.0` feature is also enabled in the `src/main/liberty/config/server.xml` file.
+A health report will be generated automatically for all services that enable MicroProfile Health. This
+feature has already been enabled for you in the `src/main/liberty/config/server.xml` file:
+
+[source, xml, role="no_copy"]
+----
+<feature>mpHealth-1.0</feature>
+----
+
+All applications must provide an implementation of the `HealthCheck` interface, which will be used to
+verify the healthiness of the services.
+
 
 // =================================================================================================
 // Adding a health check to the system service
 // =================================================================================================
 
-=== Adding a health check to the system service
+=== Adding a health check to the `system` service
+
+Before you proceed, navigate to the `start` directory where you will find the starting project:
+
+```
+cd ../start
+```
 
-Create health check class for the `system` service in the
-`src/main/java/io/openliberty/guides/system/SystemHealth.java` file:
+Implement the `HealthCheck` interface in the `src/main/java/io/openliberty/guides/system/SystemHealth.java` file:
 
 [source, java, indent=0]
 ----
 include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tags=SystemHealth]
 ----
 
-The `org.eclipse.microprofile` package contains the required MicroProfile Health Check classes.
-
-The `@Health` annotation indicates that this class reports the state of
-`system` service through `http://localhost:9080/health` URL. The `@ApplicationScoped` annotation is
-required because the health check service must stay active within the lifetime of the application
-to report the state.
+The `@Health` annotation indicates that this particular bean implements the `HealthCheck` interface.
+By pairing this annotation with the `ApplicationScoped` context from the CDI API, the bean is discovered
```

Comment:
I believe this is the first time you mentioning "CDI" in this guide, you may want to spell it out.